keadm/cmd/keadm/app/cmd/util/k8sinstaller.go

```diff
@@ -114,11 +115,19 @@ func installCRDs(ks *K8SInstTool) error {
 		return err
 	}
 
-	crds := map[string][]string{"devices": {"devices/devices_v1alpha2_device.yaml",
-		"devices/devices_v1alpha2_devicemodel.yaml"}, "reliablesyncs": {"reliablesyncs/cluster_objectsync_v1alpha1.yaml",
-		"reliablesyncs/objectsync_v1alpha1.yaml"},
-		"router": {"router/router_v1_rule.yaml",
-			"router/router_v1_ruleEndpoint.yaml"},
+	crds := map[string][]string{
+		"devices": {
+			"devices/devices_v1alpha2_device.yaml",
```

Comment:
Can be clearer: directory is unnecessary
```suggestion
			"devices_v1alpha2_device.yaml",
```